client/app/contexts/cozy/index.json

```diff
@@ -17,5 +17,49 @@
     "figure": "facture-telephonique-lg.jpg",
     "color": {"css": "#1FA8F1"},
     "important": true
+  },
+  {
+    "slug": "fai-bills",
+    "connectors": [{
+      "slug": "bouyguesbox"
+    }, {
+      "slug": "free"
+    }, {
+      "slug": "numericable"
+    }, {
+      "slug": "sfr_box"
+    }],
+    "figure": "facture-fournisseur-internet-lg.jpg",
+    "color": {"css": "#1FA8F1"},
+    "important": true
+  },
+  {
+    "slug": "contacts",
+    "connectors": [{
+      "slug": "googlecontacts"
+    }, {
+      "slug": "linkedin"
+    }],
+    "figure": "regroupement-contacts-lg.jpg",
+    "color": {"css": "#1FA8F1"},
+    "important": true
+  },
+  {
+    "slug": "agenda",
+    "connectors": [ {
+      "slug": "doctolib"
+    },{
+      "slug": "facebook_events"
+    }, {
+      "slug": "ical_feed"
+    },{
+      "slug": "meetup"
+    }, {
+      "slug": "voyages_sncf"
+    }],
+    "figure": "regroupement-agendas-lg.jpg",
+    "color": {"css": "#1FA8F1"},
+    "important": true
   }]
+
```

Comment:
“other” should really be avoided. Here are some examples of categories in which those could fit:

– Hosting: digitalocean kimsufi_ca kimsufi_eu github online_net ovh_ca ovh_eu runabove soyoustart_ca soyoustart_eu
– Transports: aprr  trainline uber voyages_sncf
– Energy: directenergie edf electrabel
– Health: malakoff_mederic
– Shops/Markets/…: materiel_net vente_privee

The only one I’m not entirely satisfied with is github, because it’s not the same kind of hosting that’s provided by others in this category.
numericable/sosh should already be in ISP/Phone.